.github/workflows/release.yaml

```diff
@@ -34,16 +42,39 @@ jobs:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
-      - name: setup pack
+      - name: Setup pack 
         uses: buildpacks/github-actions/setup-pack@2c2fe0dad1a532cdcde622787d752df8fc3172f0 # v5.1.0
+      - name: Install cosign
+        uses: sigstore/cosign-installer@main
+      - name: Install crane
+        uses: imjasonh/setup-crane@v0.1
       - name: Build and publish image
+        id: build_n_publish_image 
         run: |
           #!/usr/bin/env bash
           set -euo pipefail
-          pack build ${IMAGE}:${GITHUB_REF_NAME} --builder ${BUILDER} --buildpack ${BUILDPACK} --publish
+          VERSION=$(git describe --tags --always)
+          COMMIT=$(git rev-parse HEAD)
+          BUILD=$(date +%FT%T%z)
+          PKG=github.com/${{ github.repository }}/pkg/version
```

Comment:
This needs to match the package name in the [go.mod](https://github.com/guacsec/guac/blob/main/go.mod#L1). If someone forks it and tries to make a release, it won't get the verison/commit/date.

Reply:
good point...I'll put the repo name back